app/modules/filter/views/filterView.js

```diff
@@ -3,25 +3,44 @@ define(function(require) {
 
     View = CMS.View.extend({
         template: _.template(require("text!../templates/filterTemplate.html")),
-
         el: false,
+        events: {
+            "change input[id^='filter']": "filterClicked"
```

Comment:
please add class for input and put events on class not on such hard css selector